Example/CryptoOnramp Example/CryptoOnramp Example/CryptoOnrampExampleView.swift

```diff
@@ -85,6 +88,9 @@ struct CryptoOnrampExampleView: View {
             .navigationBarTitleDisplayMode(.inline)
         }
         .onAppear {
+            guard coordinator == nil else {
+                return
+            }
```

Comment:
I was experiencing an issue where this was getting triggered on dismissal of the identity sheet, effectively resetting verification status. Regardless, I didn’t mean for reinitialization to occur, so I prevent it.